CHANGELOG.rst

```diff
@@ -23,6 +23,7 @@ v2.1.0
 * Fix crash when running ``--porcelain show``.
 * Show ``id`` for todos everywhere (eg: including new, etc).
 * Add the ``ctrl-s`` shortcut for saving in the interactive editor.
+* Show "Today" or "Tomorrow" when due date is today or tomorrow
```

Comment:
v2.1.0 was released a couple of days ago, so you need to move this up. ^_^

Note that I try and keep things in reverse-chronological order too, so it should be at the very top.